OsmAnd/res/values/strings.xml

```diff
@@ -13,38 +13,38 @@
 -->
 
     <string name="shared_string_mini">mini</string>
-    <string name="sample_wikivoyage">Sample Wikivoyage</string>
+    <string name="sample_wikivoyage">Wikivoyage example</string>
     <string name="travel_route_types">Route types</string>
     <string name="shared_string_files">Files</string>
-    <string name="service_is_not_available_please_try_later">Service is not available, please try later</string>
-    <string name="mapillary_menu_filter_description_new">Filter images by date or by type. Only active in closeup zoom.</string>
+    <string name="service_is_not_available_please_try_later">Unavailable service. Please try again later.</string>
+    <string name="mapillary_menu_filter_description_new">Filter images by date or by type in closeup zoom.</string>
     <string name="start_point">Start</string>
-    <string name="remove_group_from_markers">Remove group from markers list</string>
-    <string name="add_group_to_markers">Add group to markers list</string>
-    <string name="routeInfo_roadClass_description">Colouring the route or track line according to road classification.</string>
-    <string name="routeInfo_surface_description">Provide additional information about the physical surface of roads/footpaths.</string>
-    <string name="routeInfo_smoothness_description">Provides a classification scheme regarding the physical usability of a way for wheeled vehicles, particularly regarding surface regularity/flatness.</string>
```

Comment:
"Provides a classification scheme" is redundant. Translators are having problems with "way".